temporian/implementation/numpy/data/event_set.py

```diff
@@ -587,3 +585,76 @@ def creator(self) -> Optional[Operator]:
         created EventSets have a `None` creator.
         """
         return self.node()._creator
+
+    def _repr_html_(self) -> str:
+        """HTML representation, mainly for IPython notebooks."""
+        features = self.schema.features[: config.max_display_features]
+        n_features = len(self.schema.features)
+        cut_features = n_features > config.max_display_features
+        indexes = self.schema.indexes
+        convert_datetime = self.schema.is_unix_timestamp
+        all_index_keys = self.get_index_keys(sort=True)
+        repr = ""
+        for index_key in all_index_keys[: config.max_display_indexes]:
+            repr += "<h3>Index: ("
```

Comment:
It would interesting to find a way to escape html characters, syntax check, and more generally, protect from injections.
There seems to be several python library for that. Alternatively, maybe we can use existing python package (e.g. xml.dom.minidom ) ?

Reply:
Alright, I'll take a look into it.